models/courier/Courier.validation.js

```diff
@@ -0,0 +1,21 @@
+import Joi from 'joi';
+import { vehicleType } from '../../constants/vehicleType';
+import { addressValidationSchema, phoneNumberPattern } from '../../helpers/validation';
+import { accountStatus } from '../../constants/accountStatus';
+
+const CourierValidationSchema = Joi.object({
+  userId: Joi.required(),
+  avatar: Joi.string().required(),
+  phoneNumber: Joi.string().pattern(phoneNumberPattern).required(),
+  address: addressValidationSchema.required(),
+  vechicleType: Joi.string(0)
+    .valid(...vehicleType)
```

Comment:
it will throw an error, `vehicleType` has invalid format for passing it to `valid(`) like this
use `...Object.values(vehicleType)` instead